platforms/posix/src/px4/common/CMakeLists.txt

```diff
@@ -37,6 +37,11 @@ add_subdirectory(lockstep_scheduler)
 
 set(EXTRA_DEPENDS)
 
+# Add in include path for fc sensor api header
+if("${PX4_BOARD}" MATCHES "voxl2")
```

Comment:
The kconfig variable should be available in cmake as well.

Reply:
Nevermind, that wouldn't be correct here anyway (board specific dir).

Reply:
I created a board specific cmake/init.cmake file with the board specific include path so that it doesn't need to be in the posix platform cmake file (as you suggested below) Thanks!